etl/steps/data/garden/ons/2025-01-21/marriages.meta.yml

```diff
@@ -0,0 +1,35 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Marriages & Divorces
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+tables:
+  marriages:
+    variables:
+      cumulative_percentage_per_100:
+        title: Proportions of men or women who had ever married by a certain age for << birth_cohort >> birth cohort
+        unit: "%"
+        short_unit: "%"
+        description_short: |-
+          The percentage of men or women born in a specific year who have ever married by a certain age.
+        description_key:
+          - Ages are presented as "exact years," meaning "by age 30" includes marriages up to the day before the 30th birthday.
```

Comment:
I think the comma should be outside of the double quotes:
"exact years,"
but I think this is not specified in the writing and style guide, so up to you. There is a similar case later with "ever married,".